test/models/test_vit.py

```diff
@@ -10,6 +10,7 @@
 @pytest.mark.parametrize('pretrained', [True])
 @pytest.mark.skip(reason="Fails on CI init")
 def test_vit(pretrained, model_class):
+    np.random.seed(seed=42)
```

Comment:
cool! can you make the seed a configurable value so it will be for sure the same and consistent in the future?

Reply:
Done